### PR TITLE
Show ITC errors on tile, as well as sequence errors

### PR DESCRIPTION
**Pull request by rpiaggio**
[sc-3107]

### CODE REVIEW

File: explore/src/main/scala/explore/Telemetry.scala [hunk 1]
```diff
@@ -95,7 +95,7 @@ object Observability:
       .withHooks[Props]
       .useEffectOnMountBy: props =>
         val sdk = new HoneycombWebSDK(props.options)
-        Callback { println(sdk); sdk.start() } *> CallbackTo(
+        Callback.log(sdk) *> Callback(sdk.start()) *> CallbackTo(
```
Comment (cquiroz) [inline explore/src/main/scala/explore/Telemetry.scala:98]: I guess we should remove the log

File: workers/src/main/scala/workers/itc/ITCGraphRequests.scala [hunk 2]
```diff
@@ -39,7 +39,7 @@ object ITCGraphRequests:
     targets:         NonEmptyList[ItcTarget],
     mode:            InstrumentRow,
     cache:           Cache[F],
-    callback:        GraphResponse => F[Unit]
+    callback:        ItcAsterismGraphResults => F[Unit]
```
Comment (cquiroz) [inline workers/src/main/scala/workers/itc/ITCGraphRequests.scala:42]: Do we need to invalidate the cache?

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/Telemetry.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/Telemetry.scala:98]: I didn't know if you wanted to leave it, the way it was before it just printed `object Object`. I'll remove.

File: workers/src/main/scala/workers/itc/ITCGraphRequests.scala (hunk 2 above)
Comment (rpiaggio) [reply workers/src/main/scala/workers/itc/ITCGraphRequests.scala:42]: This is used to send the results back to the client. Here it was just a type rename, the type was moved to `shared` into the `explore.model.itc` package.

**Pull request merged**